src/skin/options.html

```diff
@@ -161,9 +161,17 @@ <h1><span class="i18n_options_title"></span></h1>
           <span class="i18n_options_webrtc_setting"></span>
         </label>
       </div>
+      <div class="checkbox">
+        <label>
+          <input type="checkbox" id="learn_in_incognito_checkbox">
+          <span class="i18n_options_incognito_setting"></span>
+        </label>
+      </div>
       <div id="settingsSuffix">
-        <br>
+        <br />
         <span class="i18n_options_webrtc_warning"></span>
+        <br /><br />
+        <span class="i18n_options_incognito_warning"></span>
```

Comment:
Does this do the right thing in Firefox 52 (ESR) where WebRTC isn't enabled and so `"i18n_options_webrtc_warning"` above will be hidden? Looks like [all of `#settingsSuffix` gets hidden](https://github.com/EFForg/privacybadger/blob/43143a185c9074d8223fe6537a314465ccccba88/src/js/options.js#L121), so we should fix this.

Reply:
I missed this completely! Fixed now.